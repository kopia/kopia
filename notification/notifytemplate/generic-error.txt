Subject: Kopia has encountered an error during {{ .EventArgs.Operation }} on {{.Hostname}}

Operation: {{ .EventArgs.OperationDetails }}
Started:   {{ .EventArgs.StartTimestamp | formatTime }}
Finished:  {{ .EventArgs.EndTimestamp | formatTime }} ({{ .EventArgs.Duration }})

{{ .EventArgs.ErrorDetails }}

Generated at {{ .EventTime | formatTime }} by Kopia {{ .KopiaBuildVersion }}.

https://kopia.io/