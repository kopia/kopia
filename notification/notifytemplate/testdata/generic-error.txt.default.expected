Subject: Kopia has encountered an error during Some Operation on some-host

Operation: Some Operation Details
Started:   Thu, 02 Jan 2020 03:04:05 +0000
Finished:  Thu, 02 Jan 2020 03:04:06 +0000 (1s)

error details

Generated at Thu, 02 Jan 2020 03:04:05 +0000 by Kopia v0-unofficial.

https://kopia.io/